src/main/java/org/spongepowered/api/data/manipulator/generator/testing/SimpleStringCustomData.java

```diff
@@ -0,0 +1,47 @@
+package org.spongepowered.api.data.manipulator.generator.testing;
+
+import org.spongepowered.api.data.DataQuery;
+import org.spongepowered.api.data.key.Key;
+import org.spongepowered.api.data.key.KeyFactory;
+import org.spongepowered.api.data.manipulator.DataManipulator;
+import org.spongepowered.api.data.manipulator.DataManipulatorBuilder;
+import org.spongepowered.api.data.manipulator.ImmutableDataManipulator;
+import org.spongepowered.api.data.manipulator.generator.CustomDataProvider;
+import org.spongepowered.api.data.manipulator.generator.DataRegistration;
+import org.spongepowered.api.data.persistence.DataBuilder;
+import org.spongepowered.api.data.value.BaseValue;
+import org.spongepowered.api.data.value.mutable.Value;
+
+public class SimpleStringCustomData {
```

Comment:
That definitely does look a lot simpler now, thanks gabizou for all the work you're doing :)